### DISCUSSION TITLE
Configurable intro screen title?

### DISCUSSION THREAD

**Discussion by decentral1se** [charmbracelet/glow#340] [Ideas]
Following https://github.com/charmbracelet/glow/issues/329 would it be possible to configure the header on the initial `glow` screen?

![image](https://user-images.githubusercontent.com/1991377/159165072-7a79390d-ca63-4c17-b414-2a82d4ec542b.png)

For more context, I'm planning to run `glow` in a pubnix & would like to use `glow` as a documentation viewer for all users. Therefore it is better UI if I can include a header here with something else that is not the name of the tool e.g. "Welcome to the docs".

If that seems doable, I can submit the PR with a config option.

**Comment by bashbunni** [comment]
Moved to a discussion in case anyone else has the same request. The related PR is https://github.com/charmbracelet/glow/pull/331
